Make sure freetype is included properly.
Patch by Jim Ramsay.

http://bugs.gentoo.org/show_bug.cgi?id=76533

--- Sources/TTF/font_ttf.cpp
+++ Sources/TTF/font_ttf.cpp
@@ -21,7 +21,8 @@
 #include <malloc.h>
 #include "Display/Font/font_generic.h"
 
-#include <freetype/freetype.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
 
 #include "font_ttf.h"
 
